### DISCUSSION TITLE
Global obfuscation (logbook obfuscation mechanism as a global masker)

### DISCUSSION THREAD

**Discussion by Albertmoiseev** [zalando/logbook#1851] [Q&A] [answered]
Dear community,
Is it possible to use obfuscation not only for requests and responses, but also when using a logger directly ?

For example, if I have setting  json-body-fields: name it will replace this field in requests/responses. 
But if i will log the object with log.info() the field would not be obfuscated. 

Is it possible to use logbook obfuscation mechanism as a global masker?

**Comment by whiskeysierra** [comment, marked as answer]
No, it doesn't work like that. You'd need to check your logging framework/library of choice whether it provides something like that.

**Reply by Albertmoiseev** [reply]
thanks for such a prompt reply and clarification )
